jss/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.mozilla</groupId>
```

Comment:
We might want to use `org.dogtagpki` instead of `org.mozilla` to avoid conflicts in case Mozilla publishes something in the future. I think the `<groupId>` is more like a vendor ID. It doesn't have to match the actual Java package names.